core/capabilities/registry.go

```diff
@@ -199,3 +200,30 @@ func NewRegistry(lggr logger.Logger) *Registry {
 		lggr: lggr.Named("CapabilitiesRegistry"),
 	}
 }
+
+// TestMetadataRegistry is a test implementation of the metadataRegistry
+// interface. It is used when ExternalCapabilitiesRegistry is not available.
+type TestMetadataRegistry struct{}
+
+func (t *TestMetadataRegistry) LocalNode(ctx context.Context) (capabilities.Node, error) {
+	peerID := p2ptypes.PeerID{}
+	workflowDON := capabilities.DON{
+		ID:            1,
+		ConfigVersion: 1,
+		Members: []p2ptypes.PeerID{
+			peerID,
+		},
+		F:                0,
```

Comment:
I hope this won't cause any validation issues down the line. Are we ever going to look at these values in other componenets?

Reply:
> Are we ever going to look at these values in other components?

Not planned atm.

Reply:
Ah it would be nice if this was configurable: I need `f = 1` for e2e testing